src/openrct2/core/String.hpp

```diff
@@ -26,18 +26,18 @@ using u8string_view = std::basic_string_view<utf8>;
 
 using codepoint_t = uint32_t;
 
-namespace CODE_PAGE
+namespace OpenRCT2
 {
-    // windows.h defines CP_UTF8
-#undef CP_UTF8
-
-    constexpr int32_t CP_932 = 932;    // ANSI/OEM Japanese; Japanese (Shift-JIS)
-    constexpr int32_t CP_936 = 936;    // ANSI/OEM Simplified Chinese (PRC, Singapore); Chinese Simplified (GB2312)
-    constexpr int32_t CP_949 = 949;    // ANSI/OEM Korean (Unified Hangul Code)
-    constexpr int32_t CP_950 = 950;    // ANSI/OEM Traditional Chinese (Taiwan; Hong Kong SAR, PRC); Chinese Traditional (Big5)
-    constexpr int32_t CP_1252 = 1252;  // ANSI Latin 1; Western European (Windows)
-    constexpr int32_t CP_UTF8 = 65001; // Unicode (UTF-8)
-} // namespace CODE_PAGE
+    enum CodePage : int32_t
```

Comment:
Any reason for not doing it an `enum class`?

Reply:
Yes, it is used as a parameter to Windows API functions quite often.